modules/python3/src/python3module.cpp

```diff
@@ -50,7 +50,7 @@ Python3Module::Python3Module(InviwoApplication* app)
     : InviwoModule(app, "Python3")
     , pythonInterpreter_(util::make_unique<PythonInterpreter>(this))
     , pythonScriptArg_("p", "pythonScript", "Specify a python script to run at startup", false, "",
-                       "Path to the file containing the script")
+                       "python script")
```

Comment:
Still a file?

Reply:
`"python script"` refers to the short part of the argument, not its description!
Cf.
```
inviwo.exe  [-p <python script>]
                                        [--update-workspaces-in-path

-p <python script>,  --pythonScript <python script>
     Specify a python script to run at startup
```